src/ucp/core/ucp_mm.c

```diff
@@ -412,28 +412,63 @@ static ucs_status_t ucp_mem_unmap_common(ucp_context_h context, ucp_mem_h memh)
 ucs_status_t ucp_mem_map(ucp_context_h context, const ucp_mem_map_params_t *params,
                          ucp_mem_h *memh_p)
 {
-    ucs_status_t            status;
-    ucp_mem_map_params_t    mem_params;
+    ucs_status_t status;
+    void         *address;
 
     /* always acquire context lock */
     UCP_THREAD_CS_ENTER(&context->mt_lock);
 
-    mem_params = *params;
-    status = ucp_mem_map_check_and_adjust_params(&mem_params);
```

Comment:
shall we delete ucp_mem_map_check_and_adjust_params() function?

Reply:
@yosefe Deleted. Thanks!